### PR TITLE
Add linked abc again

### PR DESCRIPTION
**Pull request by tspyrou**
This is a draft for looking at the CI.

### CODE REVIEW

File: src/rmp/src/Restructure.cpp [hunk 1]
```diff
@@ -186,104 +187,56 @@ void Restructure::runABC()
...
+	       "Writing ABC script file {}.",
+	       abc_script_file);
+
+    if (writeAbcScript(abc_script_file)) {
+      // call linked abc
+      Abc_Frame_t * abc_frame;
+      Abc_Start();
+      abc_frame = Abc_FrameGetGlobalFrame();
+      std::string command = "source " + abc_script_file;
+      child_proc[curr_mode_idx] = Cmd_CommandExecute( abc_frame, command.c_str() );
```
Comment (maliberty) [inline src/rmp/src/Restructure.cpp:218]: Why do we have to write a file if it is linked (and later read a blif)?  Is that something for a future PR?
Should child_proc be abc_return_code ?  I don't think there is a child proc anymore.

File: src/rmp/src/Restructure.cpp [hunk 2]
```diff
@@ -186,104 +187,56 @@ void Restructure::runABC()
...
-        child_proc[child_idx] = 0;
-        logger_->warn(
-            RMP,
-            15,
-            "ABC failed with code {}. Please check {} log file for details.",
-            return_status,
-            logfile_ + std::to_string(child_idx));
-      }
-    }  // end wait
+    std::string abc_script_file = work_dir_name_
```
Comment (maliberty) [inline src/rmp/src/Restructure.cpp:199]: const

File: src/rmp/src/Restructure.cpp [hunk 3]
```diff
@@ -186,104 +187,56 @@ void Restructure::runABC()
...
+	       1,
+	       "Writing ABC script file {}.",
+	       abc_script_file);
+
+    if (writeAbcScript(abc_script_file)) {
+      // call linked abc
+      Abc_Frame_t * abc_frame;
+      Abc_Start();
+      abc_frame = Abc_FrameGetGlobalFrame();
+      std::string command = "source " + abc_script_file;
```
Comment (maliberty) [inline src/rmp/src/Restructure.cpp:217]: const

File: src/rmp/src/Restructure.cpp [hunk 4]
```diff
@@ -186,104 +187,56 @@ void Restructure::runABC()
...
+	       "remap",
+	       1,
+	       "Writing ABC script file {}.",
+	       abc_script_file);
+
+    if (writeAbcScript(abc_script_file)) {
+      // call linked abc
+      Abc_Frame_t * abc_frame;
+      Abc_Start();
+      abc_frame = Abc_FrameGetGlobalFrame();
```
Comment (maliberty) [inline src/rmp/src/Restructure.cpp:216]: Abc_Start();
Abc_Frame_t * abc_frame = Abc_FrameGetGlobalFrame();

**Review by maliberty — CHANGES_REQUESTED** [review]

File: src/rmp/src/Restructure.cpp (hunk 1 above)
Comment (tspyrou) [reply src/rmp/src/Restructure.cpp]: Yes, this is a future PR. I wanted to start with this milestone to get rid of the fork and get a unit test in.

**Review by maliberty — APPROVED** [review]
Please cleanup any warnings soon so that they don't clutter up the output.

**Pull request merged**